tests/user.test.ts

```diff
@@ -75,4 +75,13 @@ describe("user tests", () => {
     expect(userListResult[0].username).toBe("nake89@gmail.com")
     expect(userListResult[0].scopes.includes("root")).toBe(true)
   })
+
+  it("deleteUser_test", async () => {
+    const us = new UserService()
+    const deleteResponse: boolean = await us.deleteUser(2)
+    const userListResult: getUserListResponse[] = await us.getUserList()
+    expect(deleteResponse).toBe(true)
+    expect(Array.isArray(userListResult)).toBe(true)
+    expect(userListResult.length).toBe(1)
```

Comment:
Because the scope.test.ts is running in parallel, it could create a user before this test is run so this test would fail possibly sometimes:
```
expect(userListResult.length).toBe(1)
```
I would for now leave out the getUserList from this. And simply test if the user has been removed by
doing deleteUser again:
```
const deleteResponse: boolean = await us.deleteUser(2)
expect(deleteResponse).toBe(false)
```
That should return false because you already deletedUser(2)

Later we can do an endpoint GET user/{id}
and we can then use that to test if there is a user.

If scope.test.us becomes too much of a problem, because we want to use userListResult.length then we can put the scopes tests into this file.